packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -160,6 +110,39 @@ class StripeHelper {
     }
   }
 
+  /**
+   * Get a cached result at a cache key and regenerated it with `refreshFunction`
+   * if its expired.
+   *
+   * @param {string} cacheKey
+   * @param {() => Promise<T>} refreshFunction
+   * @returns {Promise<T | undefined>} possibly cached result
+   */
+  async cachedResult(cacheKey, refreshFunction) {
```

Comment:
To be consistent with the other function names in here, this should start with verb.